docs/sources/tempo/setup/operator/multitenancy.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Setup multi-tenancy
+description: Set up multi-tenancy
+menuTitle: Multi-tenancy
+weight: 250
+aliases:
+- /docs/tempo/operator/multitenancy
+---
+
+# Multi-tenancy
+
+Tempo is a multi-tenant distributed tracing backend. It supports multi-tenancy through the use of a header: `X-Scope-OrgID`.
+See [multi-tenancy docs]({{< relref "../../operations/multitenancy" >}}) for more details. This document outlines how to deploy and use multi-tenant Tempo with the Operator.
```

Comment:
Prefer "refer to X" over "see X".
 
 nit:  [Semantic line breaks](https://sembr.org/) are better for future line based diffing in the GitHub UI and result in the same HTML output. 
 
 ```suggestion
Tempo is a multi-tenant distributed tracing backend.
It supports multi-tenancy through the use of a header: `X-Scope-OrgID`.
For more information, refer to [Enable multitenancy]({{< relref "../../operations/multitenancy" >}}). 
This document outlines how to deploy and use multi-tenant Tempo with the Operator.
```